tests/common/test_cgroups.py

```diff
@@ -59,7 +60,8 @@ def test_correct_creation(self):
         self.assertEqual(test_cgroup.path, "dummy_path")
         self.assertEqual(test_cgroup.name, "test_extension")
 
-    def test_is_active(self):
+    @patch("azurelinuxagent.common.osutil.default.DefaultOSUtil._get_proc_stat")
```

Comment:
why is this needed? thanks

Reply:
I am adding `@patch("azurelinuxagent.common.osutil.default.DefaultOSUtil._get_proc_stat")` in tests to make them run on Mac, and those tests do not need `/proc/stat` file. In tests that actually need `/proc/stat` file, I don't patch it.

Reply:
Added the comments in the tests as well. Thanks.